packages/amplify-graphql-api-construct-tests/src/__tests__/backends/sql-models/app.ts

```diff
@@ -58,6 +58,9 @@ new AmplifyGraphqlApi(stack, 'SqlBoundApi', {
       dbConnectionConfig: {
         ...dbDetails.ssmPaths,
       },
+      sqlLambdaProvisionedConcurrencyConfig: {
```

Comment:
I can't get the information I need in order to assert on the provisioned concurrency. I would need to add an additonal CFN output for the SQL Lambda function name. Then I could use the AWS SDK to query for the provisioned concurrency status. I'm not sure adding that is worth it though.

Adding `sqlLambdaProvisionedConcurrencyConfig` here does assert that the API doesn't break when using this setting.

Reply:
Can't you `synth` and inspect the rendered template?

Reply:
Also, since you own the test app, you can add your own `CfnExport` for `api.resources.functions.[WhateverTheFunctionNameIs].functionName` if you want to go that route.